### PR TITLE
Feature: Added alerts output to capture system

### PR DESCRIPTION
**Pull request by laf**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

Added alerts to the capture system so people can run checks against rules for a device.

image

### CODE REVIEW

File: html/includes/functions.inc.php [hunk 1]
```diff
@@ -1360,3 +1360,21 @@ function get_auth_ad_group_filter($groupname)
     }
     return $group_filter;
 }
+
+/**
+ * @param $filename
+ * @param $output
+ */
+function file_download($filename, $output)
```
Comment (murrant) [inline html/includes/functions.inc.php]: I'd rename this variable to $content

Comment (laf) [inline html/includes/functions.inc.php]: Updated and pushed.

File: html/includes/functions.inc.php [hunk 2]
```diff
@@ -1343,3 +1343,41 @@ function ipmiSensorName($hardwareId, $sensorIpmi, $rewriteArray)
         return $sensorIpmi;
     }
 }
+
+function get_auth_ad_user_filter($username)
```
Comment (murrant) [inline html/includes/functions.inc.php]: Something wrong here, these shouldn't be here...

**Comment by laf** [comment]
Updated.

**Pull request merged**